src/app/about/about.component.html

```diff
@@ -1,91 +1,189 @@
+      <mat-expansion-panel>
+        <mat-expansion-panel-header>
+          Why was ABM created?
+        </mat-expansion-panel-header>
 
-  <h2>Why was ABM created?</h2>
-  <p>The ABM is a methodology designed to make benchmark creation and maintenance easier. By creating this methodology, we wish
-    to address the following concerns:</p>
-  <ul>
-    <li>Outdated benchmark suites: Well-established benchmark suites are often created by hand, and remain unchanged for years.
-      As a result, analyses and tools are tested on outdated code, which is not representative of real-world software. By
-      automatically constructing the collection, the code base can be regularly updated to provide up-to-date test projects.</li>
-    <li>Analysis/tool scope: Benchmark suites are very general, and aim to cover many different types of software. On the other
-      hand, analyses and tools are sometimes targeted at specific programs, such as Java web applications for example. Evaluating
-      them on other types of software is insubstantial. When mining GitHub, the scope of the collection can be restricted
-      to the target of the analysis/tool, and provide an adapted, yet still representative collection of up-to-date applications.
-      The current collection targets Java web applications, and the same automated mining methodology can be applied to different
-      types of target programs.</li>
-  </ul>
-  <p>For comparison and replication purposes, old versions of the collections are archived.</p>
+        <p>The ABM is a methodology designed to make benchmark creation and maintenance easier.</p>
+        <p>By creating this methodology, we wish to address the following concerns:</p>
+        <ul>
+          <li>Outdated benchmark suites: Well-established benchmark suites are often created by hand, and remain
+            unchanged
+            for years.
+            As a result, analyses and tools are tested on outdated code, which is not representative of real-world
+            software.
+            By
+            automatically constructing the collection, the code base can be regularly updated to provide up-to-date
+            test
+            projects.</li>
+          <li>Analysis/tool scope: Benchmark suites are very general, and aim to cover many different types of
+            software.
+            On
+            the other
+            hand, analyses and tools are sometimes targeted at specific programs, such as Java web applications for
+            example.
+            Evaluating
+            them on other types of software is insubstantial. When mining GitHub, the scope of the collection can
+            be
+            restricted
+            to the target of the analysis/tool, and provide an adapted, yet still representative collection of
+            up-to-date
+            applications.
+            The current collection targets Java web applications, and the same automated mining methodology can be
```

Comment:
We don't have a current collection. This was the case for the old version of ABM. Now, ABM is more of a search engine. Please rephrase that.